### DISCUSSION TITLE
mpiexec

### DISCUSSION THREAD

**Discussion by goodman249** [firemodels/fds#12639] [General]
#!/bin/bash
#PBS -N job_name
#PBS -e <pwd>/job_name.err
#PBS -o <pwd>/job_name.log
#PBS -l nodes=4:ppn=2
#PBS -l walltime=24:0:0
export OMP_NUM_THREADS=2
export I_MPI_PIN_DOMAIN=omp
cd <pwd>
mpiexec -n 8 <full_path>/fds job_name.fds

May I ask, is mpiexec using its own supercomputing system, or is it using FDS's built-in FDS-6.7.4/bin/INTEL/bin/mpiexec?

**Comment by rmcdermo** [comment]
It depends on your environment.  Type `which mpiexec` and it should tell you where it is pointing.

Also, take care to read the message at the end of the FDS installation procedure.  It tells you how to set up your environment to use mpi.
